packages/select/src/FilterMultiSelect/components/SelectionControlButton/SelectionControlButton.scss

```diff
@@ -22,7 +22,7 @@
   border-color: $border-borderless-border-color;
   border-radius: $border-solid-border-radius;
 
-  padding: 0;
+  padding: 7px 11px;
```

Comment:
Is there a way to use our design tokens? If not (which is a different problem), this should probably still be in `rem`.